sis_provisioner/dao/student.py

```diff
@@ -0,0 +1,51 @@
+from uw_person_client import UWPersonClient
+
+
+class UWHandshakeClient(UWPersonClient):
```

Comment:
No need to subclass UWPersonClient anymore. Just define and instance of UWPersonClient since you are only using the standard library.